src/components/ShoppingList/ShoppingList.js

```diff
@@ -7,6 +8,21 @@ function ShoppingList({ listId }) {
     db.collection(`lists/${listId}/items`).orderBy('purchaseInterval', 'asc'),
   );
 
+  const handleCheck = (e) => {
+    if (!e.target.checked) return; // don't do anything if the checkbox isn't checked
```

Comment:
I don't think this is needed since `handleCheck` will only fire when the checkbox is checked.